static/src/javascripts/projects/common/modules/commercial/contributions-utilities.js

```diff
@@ -499,6 +499,15 @@ const makeGoogleDocBannerVariants = (
     return variants;
 };
 
+const makeEngagementBannerVariant = (
```

Comment:
maybe call this `makeBannerABTestVariant` for consistency with the function above?